osu.Game/Screens/Play/Player.cs

```diff
@@ -346,6 +349,18 @@ public override void OnEntering(IScreen last)
                 .Delay(250)
                 .FadeIn(250);
 
+            ShowStoryboard.ValueChanged += s =>
+            {
+                if (s.NewValue && storyboard == null)
+                    initializeStoryboard(true);
+            };
+
+            Background.EnableUserDim.Value = true;
+
+            storyboardReplacesBackground.BindTo(Background.StoryboardReplacesBackground);
```

Comment:
Same thing here, but it matters more here, the `.BindTo()` direction should be inversed.